docs/release-notes/v0.7.0.rst

```diff
@@ -0,0 +1,254 @@
+  ``default`` kwarg in :func:`pyhf.set_backend`.
+  (PR :pr:`1646`)
+* :func:`pyhf.pdf.Model` now allows for both :class:`list` and ``pyhf.tensorlib`` objects
+  to exist in the ``spec``.
+  (PR :pr:`1647`)
+* The minimum required dependencies have been updated to support added features:
+
+   - ``scipy>=1.2.0`` (PR :pr:`1274`)
+   - ``click>=8.0.0`` (PRs :pr:`1909`, :pr:`1958`)
+   - ``jsonschema>=4.15.0`` (PRs :pr:`1976`, :pr:`1979`)
+   - ``importlib_resources>=1.4.0`` (for Python 3.7, 3.8) (PR :pr:`1979`)
+   - ``typing_extensions>=3.7.4.3`` (for Python 3.7 only) (PRs :pr:`1940`, :pr:`1961`)
+
+* The minimum required backend versions have been updated to support added features:
+
+   - JAX backend requires ``jax>=0.2.10``, ``jaxlib>=0.1.61`` (PR :pr:`1962`)
+   - PyTorch backend requires ``torch>=1.10.0`` (PR :pr:`1657`)
+   - TensorFlow backend requires ``tensorflow>=2.7.0``, ``tensorflow-probability>=0.11.0`` (PRs :pr:`1962`, :pr:`1657`)
+   - iminuit optimizer requires ``iminuit>=2.7.0`` (PR :pr:`1895`)
+   - xmlio extra requires ``uproot>=4.1.1`` (PR :pr:`1567`)
+
+Fixes
+-----
+
+* Use improvements to ``jsonschema.RefResolver`` to avoid
+  ``jsonschema.exceptions.RefResolutionError``.
+  (PR :pr:`1976`)
+
+* Use the conditional maximum likelihood estimators of the nuisance parameters
+  to create the sampling distributions for :class:`pyhf.infer.calculators.ToyCalculator`.
+  (PR :pr:`1610`)
+  This follows the joint recommendations of the ATLAS and CMS experiments in
+  |LHC Higgs search combination procedure|_.
+
+Features
+--------
+
+Python API
+~~~~~~~~~~
+
+* The following functions have been added to the ``pyhf.tensorlib`` API:
+
+
+   - ``pyhf.tensorlib.transpose`` (PR :pr:`1696`)
+   - ``pyhf.tensorlib.percentile`` (PR :pr:`817`)
+
+* :func:`pyhf.readxml.parse` now supports reading of XML configurations with absolute paths
+  with the addition of the ``mounts`` optional argument.
+  (PR :pr:`1909`)
+
+* Add support for overriding the paths for finding schemas, using the ``pyhf``
+  installed location as a base via ``pyhf.utils.schemas``.
+  (PRs :pr:`1753`, :pr:`1818`)
+
+  .. code:: pycon
+
+      >>> from pathlib import Path
+      >>> import pyhf.schema
+      >>> current_schema_path = pyhf.schema.path
+      >>> current_schema_path
+      PosixPath('/path/to/your/venv/lib/python3.X/site-packages/pyhf/schemas')
+      >>> custom_schema_path = Path("/path/to/custom/pyhf/schema")
+      >>> with pyhf.schema(custom_schema_path):
+      ...     print(repr(pyhf.schema.path))
+      ...
+      PosixPath('/path/to/custom/pyhf/schema')
+      >>> pyhf.schema.path
+      PosixPath('/path/to/your/venv/lib/python3.X/site-packages/pyhf/schemas')
+
+* In :func:`pyhf.workspace.Workspace.model` the parameter of interest specified
+  in the measurement may now be overridden using the added ``poi_name`` kwarg.
+  (PR :pr:`1636`)
+
+* Support for writing user custom modifiers is added.
+  (PRs :pr:`1625`, :pr:`1644`)
```

Comment:
It would be nice to show something like a minimal example for the benefits of PR #1644, but I'm not actually sure how much code this would take.

@lukasheinrich what are your thoughts? If adding a code example is too much here can you suggest some technical details to expand on?

Reply:
The gist in https://github.com/scikit-hep/pyhf/issues/850#issuecomment-1239975121 is a good (not quite minimal) example I think. Before #1991 I would view this as power user territory and the interested people may already be aware of it through #850 anyway.

Reply:
@alexander-held I've added a comment on "targeted at expert users" now.